script/add_podcast_to_show_series.rb

```diff
@@ -0,0 +1,15 @@
+require 'csv'
+
+csv_path = "/tmp/missing_podcasts.csv"
+
+CSV.foreach(csv_path, headers: true) do |row|
+  # show_title = row[:show_series]
+  slug = row[:show_series_slug]
+  show_series = ShowSeries.friendly.find slug
+  track = Track.find_by(title: row[:title])
+  if show_series
```

Comment:
this should check for `track` too. there's quite a few records in that csv that have no title